integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java

```diff
@@ -0,0 +1,505 @@
+
+    // install and verify Elasticsearch
+    logger.info("install and verify Elasticsearch");
+    elasticsearchParams = assertDoesNotThrow(() -> installAndVerifyElasticsearch(),
+            String.format("Failed to install Elasticsearch"));
+    assertTrue(elasticsearchParams != null, "Failed to install Elasticsearch");
+
+    // install and verify Kibana
+    logger.info("install and verify Kibana");
+    kibanaParams = assertDoesNotThrow(() -> installAndVerifyKibana(),
+        String.format("Failed to install Kibana"));
+    assertTrue(kibanaParams != null, "Failed to install Kibana");
+
+    // install and verify Operator
+    installAndVerifyOperator(opNamespace, opNamespace + "-sa",
+        false, 0, true, domainNamespace);
+
+    // create fluentd configuration
+    configFluentd();
+
+    // create and verify WebLogic domain image using model in image with model files
+    String imageName = createAndVerifyDomainImage();
+
+    // create and verify one cluster domain
+    logger.info("Create domain and verify that it's running");
+    createAndVerifyDomain(imageName);
+
+    testVarMap = new HashMap<String, String>();
+
+    StringBuffer elasticsearchUrlBuff =
+        new StringBuffer("curl http://")
+            .append(ELASTICSEARCH_HOST)
+            .append(":")
+            .append(ELASTICSEARCH_HTTP_PORT);
+    k8sExecCmdPrefix = elasticsearchUrlBuff.toString();
+    logger.info("Elasticsearch URL {0}", k8sExecCmdPrefix);
+
+    // Verify that ELK Stack is ready to use
+    testVarMap = verifyLoggingExporterReady(opNamespace, null, FLUENTD_INDEX_KEY);
+    Map<String, String> kibanaMap = verifyLoggingExporterReady(opNamespace, null, KIBANA_INDEX_KEY);
+
+    // merge testVarMap and kibanaMap
+    testVarMap.putAll(kibanaMap);
+  }
+
+  /**
+   * Uninstall ELK Stack and delete domain custom resource.
+   */
+  @AfterAll
+  void tearDown() {
+    if (System.getenv("SKIP_CLEANUP") == null
+        || (System.getenv("SKIP_CLEANUP") != null
+        && System.getenv("SKIP_CLEANUP").equalsIgnoreCase("false"))) {
+      // uninstall ELK Stack
+      if (elasticsearchParams != null) {
+        logger.info("Uninstall Elasticsearch pod");
+        assertDoesNotThrow(() -> uninstallAndVerifyElasticsearch(elasticsearchParams),
+            "uninstallAndVerifyElasticsearch failed with ApiException");
+      }
+
+      if (kibanaParams != null) {
+        logger.info("Uninstall Elasticsearch pod");
+        assertDoesNotThrow(() -> uninstallAndVerifyKibana(kibanaParams),
+            "uninstallAndVerifyKibana failed with ApiException");
+      }
+
+      // delete domain custom resource
+      logger.info("Delete domain custom resource in namespace {0}", domainNamespace);
+      assertDoesNotThrow(() -> deleteDomainCustomResource(domainUid, domainNamespace),
+          "deleteDomainCustomResource failed with ApiException");
+      logger.info("Deleted Domain Custom Resource " + domainUid + " from " + domainNamespace);
```

Comment:
do we need it here? CleanupUtil cleans all generated namespaces

Reply:
removed